src/app/options/options.component.spec.ts

```diff
@@ -49,10 +49,13 @@ describe('OptionsComponent', () => {
       selector: '.inputInput'}));
 
     await input.focus();
-    let options = await input.getOptions();
 
-    expect(options.length).toEqual(2);
-    expect((await options[1].getText())).toEqual("test2 [Read-only]");
+    component.inputList.subscribe(
```

Comment:
Instead of subscribing directly, can you make use of 'toHaveBeenCalledWith' ....
expect(componet.inputList).toHaveBeenCalledWith([''test2 [Read-only], ''])